### ISSUE TITLE
Now Playing player scrubber is not visible to voice over

### ISSUE THREAD

**Issue by bjtitus** [Automattic/pocket-casts-ios#1315]
## Expected Behavior

Scrubber should be read and interactive using Voice Over.

## Actual Behavior

Scrubber is not in the controls listed by Voice Over

### Step-by-step reproduction instructions

1. Go to Now Playing screen in Pocket Casts
2. Enable Voice Over (can use Siri to enable)
3. Swipe through controls
4. Notice that scrubber is not included in controls list

### Did you search for existing bug reports?

Initially reported in #7470416-zen

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

iPhone 15 Pro Max, iOS 17.2.1, Pocket Casts 7.5.4
